tools/layer-zero-example/README.md

```diff
@@ -0,0 +1,175 @@
+# Layer Zero examples
+
+### OApp
```

Comment:
You should add a bit of a description to each section or ahead of the commands so it's clear what you're doing in the setup and what the test runs cover

Reply:
Added.